gnomad_mitochondria/utils/annotations.py

```diff
@@ -5,6 +5,10 @@
 
 from gnomad.resources.grch38.gnomad import POPS
 from gnomad.resources.grch38.reference_data import dbsnp, _import_dbsnp
+from gnomad.utils.annotations import age_hists_expr
+from gnomad.utils.reference_genome import add_reference_sequence
+from gnomad_qc.v3.resources.meta import meta  # pylint: disable=import-error
```

Comment:
was confused for a second but now I see where these are used in this file - just wasn't added in this commit